client/ayon_flame/plugins/create/create_shot_clip.py

```diff
@@ -1,6 +1,9 @@
 from copy import deepcopy
+from ayon_core.lib import Logger
 import ayon_flame.api as ayfapi
 
+log = Logger.get_logger(__name__)
```

Comment:
Is there reason to create different logger from create plugin logger?

Reply:
the build in self.log is not printing anywhere so this was the only way to print anything

Reply:
The built-in logger is exactly the same what you do here.

Reply:
Fixed perhaps it was not working before dure debug type of logger.